### PR TITLE
[CLI] update Parameter Store reference for v11

### PR DESCRIPTION
**Pull request by jhockett**
#### Description of changes:
Adding instructions for creating parameters in Parameter Store for Amplify projects.

### Instructions

**If this PR should not be merged upon approval for any reason, please submit as a DRAFT**

Which product(s) are affected by this PR (if applicable)?
- [x] amplify-cli
- [ ] amplify-ui
- [ ] amplify-studio
- [ ] amplify-hosting
- [ ] amplify-libraries

Which platform(s) are affected by this PR (if applicable)?
- [ ] JS
- [ ] iOS
- [ ] Android
- [ ] Flutter
- [ ] React Native

**Please add the product(s)/platform(s) affected to the PR title**

#### Checks

- [x] Does this PR conform to [the styleguide](https://github.com/aws-amplify/docs/blob/main/STYLEGUIDE.md)?

- [ ] Does this PR include filetypes other than markdown or images? Please add or update unit tests accordingly.

- [ ] Are any files being deleted with this PR? If so, have the needed redirects been created?

- [ ] Are all links in MDX files using the MDX link syntax rather than HTML link syntax? <br />
      _ref: MDX: `[link](https://link.com)`
            HTML: `<a href="https://link.com">link</a>`_
            
### When this PR is ready to merge, please check the box below
- [ ] Ready to merge

_By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license._

### CODE REVIEW

File: src/pages/cli/reference/ssm-parameter-store.mdx [hunk 1]
```diff
@@ -12,3 +12,29 @@ The `team-provider-info.json` file (from here on referred to as the TPI file) is
 With this change the values historically stored in the TPI file are now being copied to Parameter Store. We are not removing the TPI file, Parameter Store is being used as a backup. If Amplify CLI needs a value that is not present in the TPI file, it will fetch the value from Parameter Store. The values in Parameter Store are updated after a successful Amplify deployment.
 
 Developers can now seamlessly move between various environments like different developer machines and even to AWS Amplify Studio.
+
+<Callout>Note: When an Amplify environement is deleted, all parameters under that environment name are deleted in Parmater Store. When an Amplify project is deleted, all parameters for that appId are deleted along with it.</Callout>
```
Comment (josefaidt) [inline src/pages/cli/reference/ssm-parameter-store.mdx:16]:
```suggestion
<Callout>Note: When an Amplify environement is deleted, all parameters under that environment name are deleted in Paramater Store. When an Amplify project is deleted, all parameters for that `appId` are deleted along with it.</Callout>
```

File: src/pages/cli/reference/ssm-parameter-store.mdx [hunk 2]
```diff
@@ -12,3 +12,29 @@ The `team-provider-info.json` file (from here on referred to as the TPI file) is
 With this change the values historically stored in the TPI file are now being copied to Parameter Store. We are not removing the TPI file, Parameter Store is being used as a backup. If Amplify CLI needs a value that is not present in the TPI file, it will fetch the value from Parameter Store. The values in Parameter Store are updated after a successful Amplify deployment.
 
 Developers can now seamlessly move between various environments like different developer machines and even to AWS Amplify Studio.
+
+<Callout>Note: When an Amplify environement is deleted, all parameters under that environment name are deleted in Paramater Store. When an Amplify project is deleted, all parameters for that `appId` are deleted along with it.</Callout>
```
Comment (jhockett) [inline src/pages/cli/reference/ssm-parameter-store.mdx:16]:
```suggestion
<Callout>Note: When an Amplify environment is deleted, all parameters under that environment name are deleted in Parameter Store. When an Amplify project is deleted, all parameters for that `appId` are deleted along with it.</Callout>
```

File: src/pages/cli/reference/ssm-parameter-store.mdx [hunk 3]
```diff
@@ -12,3 +12,29 @@ The `team-provider-info.json` file (from here on referred to as the TPI file) is
...
+
+![image](/images/cli/parameter-store/view-parameter-store.png)
+
+Amplify parameter keys must follow a strict naming pattern:
+
+/amplify/<appId>/<environment-name>/AMPLIFY_<category-name>_<resource-name>_<parameter-name>
+
+Only Standard parameters with the default String type are supported.
+
+![image](/images/cli/parameter-store/create-parameter.png)
```
Comment (josefaidt) [inline src/pages/cli/reference/ssm-parameter-store.mdx:38]:
```suggestion
![AWS SSM Parameter Store's "Create Parameter" form with Amplify's parameter naming pattern](/images/cli/parameter-store/create-parameter.png)
```

File: src/pages/cli/reference/ssm-parameter-store.mdx [hunk 4]
```diff
@@ -12,3 +12,29 @@ The `team-provider-info.json` file (from here on referred to as the TPI file) is
...
+
+![image](/images/cli/parameter-store/select-parameter-store.png)
+
+With Parameter Store opened, select "Create Parameter".
+
+![image](/images/cli/parameter-store/view-parameter-store.png)
+
+Amplify parameter keys must follow a strict naming pattern:
+
+/amplify/<appId>/<environment-name>/AMPLIFY_<category-name>_<resource-name>_<parameter-name>
```
Comment (josefaidt) [inline src/pages/cli/reference/ssm-parameter-store.mdx:34]:
```suggestion
```
/amplify/<appId>/<environment-name>/AMPLIFY_<category-name>_<resource-name>_<parameter-name>
```
```

File: src/pages/cli/reference/ssm-parameter-store.mdx [hunk 5]
```diff
@@ -12,3 +12,29 @@ The `team-provider-info.json` file (from here on referred to as the TPI file) is
...
+
+![image](/images/cli/parameter-store/select-ssm-console.png)
+
+Once the SSM console has been opened, navigate to Parameter Store.
+
+![image](/images/cli/parameter-store/select-parameter-store.png)
+
+With Parameter Store opened, select "Create Parameter".
+
+![image](/images/cli/parameter-store/view-parameter-store.png)
```
Comment (josefaidt) [inline src/pages/cli/reference/ssm-parameter-store.mdx:30]:
```suggestion
![AWS SSM Parameter Store's landing page with a list of parameters](/images/cli/parameter-store/view-parameter-store.png)
```

File: src/pages/cli/reference/ssm-parameter-store.mdx [hunk 6]
```diff
@@ -12,3 +12,29 @@ The `team-provider-info.json` file (from here on referred to as the TPI file) is
...
+
+## Manually creating parameters
+
+In some multi-environment scenarios, it is helpful to create parameters in Parameter Store manually. To do so, head to the AWS Console and search for SSM.
+
+![image](/images/cli/parameter-store/select-ssm-console.png)
+
+Once the SSM console has been opened, navigate to Parameter Store.
+
+![image](/images/cli/parameter-store/select-parameter-store.png)
```
Comment (josefaidt) [inline src/pages/cli/reference/ssm-parameter-store.mdx:26]:
```suggestion
![AWS Systems Manager side navigation with "Parameter Store" highlighted](/images/cli/parameter-store/select-parameter-store.png)
```

File: src/pages/cli/reference/ssm-parameter-store.mdx [hunk 7]
```diff
@@ -12,3 +12,29 @@ The `team-provider-info.json` file (from here on referred to as the TPI file) is
...
 
 Developers can now seamlessly move between various environments like different developer machines and even to AWS Amplify Studio.
+
+<Callout>Note: When an Amplify environment is deleted, all parameters under that environment name are deleted in Parameter Store. When an Amplify project is deleted, all parameters for that `appId` are deleted along with it.</Callout>
+
+## Manually creating parameters
+
+In some multi-environment scenarios, it is helpful to create parameters in Parameter Store manually. To do so, head to the AWS Console and search for SSM.
+
+![image](/images/cli/parameter-store/select-ssm-console.png)
```
Comment (josefaidt) [inline src/pages/cli/reference/ssm-parameter-store.mdx:22]:
```suggestion
![AWS Console's search with "SSM"](/images/cli/parameter-store/select-ssm-console.png)
```

**Review by josefaidt — DISMISSED** [review]

**Review by josefaidt — APPROVED** [review]

**Review by kuhlmanp — APPROVED** [review]

**Review by katiegoines — APPROVED** [review]

**Pull request merged**